var/spack/repos/builtin/packages/simgrid/package.py

```diff
@@ -11,32 +11,56 @@ class Simgrid(CMakePackage):
     Clouds, HPC or P2P systems."""
 
     homepage = "http://simgrid.org/"
-    url      = "https://gforge.inria.fr/frs/download.php/file/37758/SimGrid-3.21.tar.gz"
-    git      = 'https://scm.gforge.inria.fr/anonscm/git/simgrid/simgrid.git'
+    url      = "https://github.com/simgrid/simgrid/releases/download/v3.27/simgrid-3.27.tar.gz"
+    git      = 'https://framagit.org/simgrid/simgrid.git'
 
+    version('3.27', sha256='51aeb9de0434066e5fec40e785f5ea9fa934afe7f6bfb4aa627246e765f1d6d7',
+            url='https://github.com/simgrid/simgrid/releases/download/v3.27/simgrid-3.27.tar.gz')
+    version('3.26', sha256='ac50da1eacc5a53b094a988a8ecde09962c29320f346b45e74dd32ab9d9f3e96',
+            url='https://github.com/simgrid/simgrid/releases/download/v3.26/simgrid-3.26.tar.gz')
```

Comment:
We can possibly drop the `url` argument in the latest version directives:
```suggestion
    version('3.27', sha256='51aeb9de0434066e5fec40e785f5ea9fa934afe7f6bfb4aa627246e765f1d6d7')
    version('3.26', sha256='ac50da1eacc5a53b094a988a8ecde09962c29320f346b45e74dd32ab9d9f3e96')
```
and use the default `url`.